packages/hoppscotch-common/assets/scss/styles.scss

```diff
@@ -559,3 +559,8 @@ details[open] summary .indicator {
   @apply rounded;
   @apply border-0;
 }
+
+// it fixes the overflow issue on flex items
+.flex {
+  min-width: 0;
+}
```

Comment:
[`flex`](https://windicss.org/utilities/layout/flexbox.html#flex) is a valid utility class in WindiCSS. I don't recommend overriding it.

You can look into adding [`min-w-0`](https://windicss.org/utilities/layout/sizing.html#min-width) class to any component that makes the layout and styling of the request tabs to stay consistent.

Reply:
Yeah, we can add that to any class. But, we have to do to everywhere we are using flex. 

Also, we are actually not overriding the class's properties. We are adding `min-width: 0` which is a new property. 

What's your thought?

Reply:
Hmm, that's not the point.
We don't need to add `min-width: 0` to all `flex` classes. Because:
1. I don't think having min-width set to zero on all `flex` classes is fixing this particular issue.
2. You can't add a new property to a WindiCSS/TailwindCSS utility classes, this goes against the CSS framework model.

Instead there's this one container in `default.vue` which this property effects the outcome.

Here's the fix:

`default.vue`

<img width="1178" alt="Screenshot 2023-05-04 at 9 44 10 PM" src="https://user-images.githubusercontent.com/10395817/236262531-f5718ea0-c5ad-414c-b152-a7156dfb9410.png">

You can revert the changes made to `styles.scss`.